README.md

```diff
@@ -26,8 +26,9 @@ allprojects {
 dependencies {
 
     ...
+        implementation 'com.github.kinecosystem.kin-sdk-android:kin-sdk-lib:1.0.5'
```

Comment:
is it before or after b&r changes (why versions were not updated already?)

Reply:
Its after, I just forgot to update the docs